third_party/2and3/flask/views.pyi

```diff
@@ -0,0 +1,22 @@
+# Stubs for flask.views (Python 3.6)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from .globals import request
+from typing import Any
+
+http_method_funcs: Any
+
+class View:
+    methods: Any = ...
+    provide_automatic_options: Any = ...
+    decorators: Any = ...
+    def dispatch_request(self) -> None: ...
+    @classmethod
+    def as_view(cls, name: Any, *class_args: Any, **class_kwargs: Any): ...
+
+class MethodViewType(type):
+    def __init__(cls, name: Any, bases: Any, d: Any) -> None: ...
+
+class MethodView:
+    def dispatch_request(self, *args: Any, **kwargs: Any): ...
```

Comment:
There are few errors on classes that inherit from this one that they don't have `as_view` class method. I suppose it is also defined here, not only on `View` above.

Reply:
Yes, this is because View is a metaclass of MethodView, can you help me on the typing here please?